hawtjni-runtime/src/main/java/org/fusesource/hawtjni/runtime/Library.java

```diff
@@ -20,27 +20,36 @@
  * 
  * It will search for the library in order at the following locations:
  * <ol>
- * <li> in the custom library path: If the "<code>library.${name}.path</code>" System property is set to a directory 
+ * <li> in the custom library path: If the "<code>library.${name}.path</code>" System property is set to a directory,
+ * subdirectories are searched:
+ *   <ol>
+ *   <li> "<code>${platform}/${arch}</code>"
+ *   <li> "<code>${platform}</code>"
+ *   <li> "<code>${os}</code>"
+ *   <li> "<code></code>"
+ *   </ol>
+ *   for 2 namings of the library:
  *   <ol>
  *   <li> as "<code>${name}-${version}</code>" library name if the version can be determined.
  *   <li> as "<code>${name}</code>" library name
  *   </ol>
  * <li> system library path: This is where the JVM looks for JNI libraries by default.
  *   <ol>
+ *   <li> as "<code>${name}${bit-model}-${version}</code>" library name if the version can be determined.
  *   <li> as "<code>${name}-${version}</code>" library name if the version can be determined.
  *   <li> as "<code>${name}</code>" library name
  *   </ol>
  * <li> classpath path: If the JNI library can be found on the classpath, it will get extracted
  * and then loaded. This way you can embed your JNI libraries into your packaged JAR files.
  * They are looked up as resources in this order:
  *   <ol>
- *   <li> "<code>META-INF/native/${platform}/${arch}/${library}</code>": Store your library here if you want to embed
+ *   <li> "<code>META-INF/native/${platform}/${arch}/${library[-version]}</code>": Store your library here if you want to embed
```

Comment:
How is this version necessary for our usecase? Especially because only one version of the native library should exist once in the classpath OR the JAR from central contains its version anyway.